libertai_client/commands/agent.py

```diff
@@ -191,16 +203,34 @@ async def deploy(
                 response_data = UpdateAgentResponse(**json.loads(await response.text()))
                 if len(response_data.error_log) > 0:
                     # Errors occurred
-                    err_console.print(f"[red]Error log:\n{response_data.error_log}")
-                    warning_text = "Some errors occurred during the deployment, please check the logs above and make sure your agent is running correctly. If not, try to redeploy it and contact the LibertAI team if the issue persists."
-                    rich.print(f"[yellow]{warning_text}")
+                    match format:
+                        case "json":
+                            json_data = f'{{"success": false, "message": "{response_data.error_log}"}}'
+                            json_object = json.loads(json_data)
+                            json_formatted_str = json.dumps(json_object, indent=2)
+                            rich.print(json_formatted_str)
```

Comment:
not sure but I think something like this would work and is simpler

```suggestion
                            json_data = {"success": False, "message": response_data.error_log}
                            json_formatted_str = json.dumps(json_object, indent=2)
                            rich.print(json_formatted_str)
```